_meta/ecs-migration.yml

```diff
@@ -117,53 +120,44 @@
 - from: context.service.version
   to: service.version
 
-#- from: context.request.url.full
-#  to: url.full
+- from: context.request.url.full
+  to: url.full
 
-#
-#- from: context.request.url.hash
-#  to: url.fragment
+- from: context.request.url.hash
+  to: url.fragment
 
-#
-#- from: context.request.url.hostname
-#  to: url.domain
+- from: context.request.url.hostname
+  to: url.domain
 
-#
-#- from: context.request.url.pathname
-#  to: url.path
+- from: context.request.url.pathname
+  to: url.path
 
-#
-#- from: context.request.url.port
-#  to: url.port
+- from: context.request.url.port
+  to: url.port
 
-#
-#- from: context.request.url.raw
-#  to: url.original
-#  index: false
-#
-#- from: context.request.url.search
-#  to: url.query
+- from: context.request.url.raw
+  to: url.original
+  index: false
 
-#
-#- from: context.request.url.protocol
-#  to: url.scheme
+- from: context.request.url.search
+  to: url.query
 
-#
-#- from: context.response.finished
-#  to: http.response.finished
+- from: context.request.url.protocol
+  to: url.scheme
 
-#
-#- from: context.response.status_code
-#  to: http.response.status_code
+- from: context.response.finished
```

Comment:
this is not necessarily an `http` response

Reply:
Apparently it only set here: https://github.com/elastic/apm-agent-nodejs/blob/8b66e282044a7a27a0e204503d3b10ba4b73a19e/lib/parsers.js#L142-L145

So should be fine

Reply:
Thanks for checking that 👍